src/ibm/hypervisor-network-mgr-src/hyp_ethernet_interface.hpp

```diff
@@ -68,8 +68,22 @@ class HypEthInterface : public CreateIface
         HypEthernetIntf::interfaceName(intfName);
 
         createIPAddressObjects();
+        watchBaseBiosTable();
     };
 
+    /* @brief Method to return the value of the input attribute
+     *        from the BaseBIOSTable
+     *  @param[in] attrName - name of the bios attribute
+     *  @param[out] - value of the bios attribute
+     */
+    std::string getAttrFromBiosTable(std::string attrName);
```

Comment:
const reference missing for the argument.

Reply:
Done